### PR TITLE
Feat/metadata

### PR DESCRIPTION
**Pull request by mheggelund**

### CODE REVIEW

File: src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx [hunk 1]
```diff
@@ -24,50 +23,81 @@ export const ModelMetadataView = () => {
...
+                    ?.filter((m) => m.metadataType === 'Field')
+                    .map((m) => (
+                      <Typography key={m.metadataId}>
+                        {m.value + ', '}
+                      </Typography>
+                    ))
+                ) : (
+                  <Typography> - No Field selected - </Typography>
+                )}
+              </Table.Cell>
```
Comment (Sinrefvol) [inline src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx:47]: Should be a reusable component, considering you are using almost the same code 3 times in total

File: src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx [hunk 2]
```diff
@@ -24,50 +23,81 @@ export const ModelMetadataView = () => {
...
+                    ?.filter((m) => m.metadataType === 'Formation')
+                    .map((m) => (
+                      <Typography key={m.metadataId}>
+                        {m.value + ',   '}
+                      </Typography>
+                    ))
+                ) : (
+                  <Typography> - No Formation selected - </Typography>
+                )}
+              </Table.Cell>
```
Comment (Sinrefvol) [inline src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx:65]: Reusable component

File: src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx [hunk 3]
```diff
@@ -24,50 +23,81 @@ export const ModelMetadataView = () => {
...
+                  .length ? (
+                  data.data.metadata
+                    ?.filter((m) => m.metadataType === 'Zone')
+                    .map((m) => (
+                      <Typography key={m.metadataId}>{m.value}</Typography>
+                    ))
+                ) : (
+                  <Typography> - No Zone selected - </Typography>
+                )}
+              </Table.Cell>
```
Comment (Sinrefvol) [inline src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx:92]: Reusable component

File: src/pages/Browse/Browse.tsx [hunk 4]
```diff
@@ -72,8 +114,64 @@ export const Browse = () => {
...
+  ) {
+    const metadataList: AddMetadataDto[] = [];
+    const analougueList: AddAnalogueDto[] = [];
+
+    metadata.zone?.forEach((item) => {
+      const obj = {
+        metadataId: item.metadataId,
+      };
+      metadataList.push(obj);
+    });
```
Comment (Sinrefvol) [inline src/pages/Browse/Browse.tsx:129]: use [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function

Comment (Sinrefvol) [reply src/pages/Browse/Browse.tsx:129]: ex: 
```js
var zones = metadata.zone?.map(x=>({ metadataId: x.metadataId }));
metadataList.push(...zones)
```

File: src/pages/Browse/Browse.tsx [hunk 5]
```diff
@@ -72,8 +114,64 @@ export const Browse = () => {
...
+
+    metadata.analogue?.forEach((item) => {
+      const obj = {
+        analogueId: item.analogueId,
+      };
+      analougueList.push(obj);
+    });
```
Comment (Sinrefvol) [inline src/pages/Browse/Browse.tsx:149]: Same as previous. Make a reusable function that can be used for all situations.

**Review by Sinrefvol — CHANGES_REQUESTED** [review]
In some situations it is fine to start turning TS lint rules off, because one rule does not fit all the situations.

However, before turning off `/* eslint-disable max-lines */` and
`/* eslint-disable max-lines-per-function */`, think what can be improved here. Sometimes, things can be rewritten in a cleaner way, or written to be reusable functions/components

File: src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx (hunk 2 above)
Comment (mheggelund) [reply src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx:65]: Fixed

File: src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx (hunk 3 above)
Comment (mheggelund) [reply src/features/ModelView/ModelMetadataView/ModelMetadataView.tsx:92]: Fixed

File: src/pages/Browse/Browse.tsx (hunk 4 above)
Comment (mheggelund) [reply src/pages/Browse/Browse.tsx:129]: Agree! Fixed.

File: src/pages/Browse/Browse.tsx (hunk 5 above)
Comment (mheggelund) [reply src/pages/Browse/Browse.tsx:149]: Fixed.

**Review by Sinrefvol — APPROVED** [review]

**Pull request merged**